replicate/training.py

```diff
@@ -307,9 +307,9 @@ def create(  # type: ignore
 
     async def async_create(
         self,
-        model: Union[str, Tuple[str, str], "Model"],
-        version: Union[str, Version],
-        input: Dict[str, Any],
+        model: Optional[Union[str, Tuple[str, str], "Model"]] = None,
+        version: Optional[Union[str, Version]] = None,
```

Comment:
I don't think `version` is optional, is it? It looks like we'll raise a `ValueError` if you don't pass it in every case.